Examples/Swift/ViewController.swift

```diff
@@ -53,6 +55,23 @@ class ViewController: UIViewController, MGLMapViewDelegate, CLLocationManagerDel
 
         simulationButton.isSelected = true
         startButton.isEnabled = false
+        
+        alertController = UIAlertController(title: "Start Navigation", message: "Select the navigation type", preferredStyle: .actionSheet)
+        alertController.addAction(UIAlertAction(title: "Default UI", style: .default, handler: { (action) in
```

Comment:
If building up the actions is slow, as claimed above and in #455, then we’re shifting that performance hit to startup, which in principle would hurt perceived performance. Did you determine that `addAction(_:)` is the problem by profiling?